### DISCUSSION TITLE
Relating e-mail attachments to the same records as the e-mail itself

### DISCUSSION THREAD

**Discussion by diderich** [YetiForceCompany/YetiForceCRM#15656] [Feature improvement]
Currently, if an e-mail is received it is related to the sender & receiver of the e-mail, if they exist in the CRM system.

In addition, if the subject contains the number of a Lead, Opportunity, Project, or ServiceContract, it is attached to those entitities as well.

It would be great if documents attached to an e-mail would be attached to the same records as the e-mail itself. Currently, the user has to manually relate attached documents to the corresponding record, which is a tedious task

**Comment by DarioGui** [comment]
Yes, and I also add another aspect (even if I don't understand if it's my mistake).
When a ticket is assigned to another user, this user cannot see the emails associated with the ticket because they are linked to the account that received the emails.
I add that I could not find a way to manually modify the email by assigning it to the other user

**Reply by diderich** [reply]
I think if the email is associated with a ticket, any user with access to the ticket should be able to see the email.
You may relate on email to an employee and than that employee can see it as related email.

**Reply by DarioGui** [reply]
I tried.
I automatically create a ticket with mailscanner, I open it and maybe modify it (usually adding comments) and then I transfer the ownership to another user.
The other user can see the ticket .. the comments .. but not the matched emails ..
even in the email, if I enter the details, I cannot change the property to show it to the new assigned user.
Thank you so much for the feedback!

**Comment by s2b-git** [comment]
Mail scanner - options for improvement #15170
